nxc/protocols/ftp/db_navigator.py

```diff
@@ -1,46 +1,53 @@
 #!/usr/bin/env python3
-# -*- coding: utf-8 -*-
 
 from nxc.nxcdb import DatabaseNavigator, print_table, print_help
 
 
 class navigator(DatabaseNavigator):
     def display_creds(self, creds):
-        data = [[
-            "CredID",
-            "Total Logins",
-            "Username",
-            "Password",
-        ]]
+        data = [
+            [
+                "CredID",
+                "Total Logins",
+                "Username",
+                "Password",
+            ]
+        ]
 
         for cred in creds:
             total_users = self.db.get_loggedin_relations(cred_id=cred[0])
-            data.append([
-                cred[0],
-                str(len(total_users)) + " Host(s)",
-                cred[1],
-                cred[2],
-            ])
+            data.append(
+                [
+                    cred[0],
+                    str(len(total_users)) + " Host(s)",
+                    cred[1],
+                    cred[2],
+                ]
+            )
         print_table(data, title="Credentials")
 
     def display_hosts(self, hosts):
-        data = [[
-            "HostID",
-            "Total Users",
-            "Host",
-            "Port",
-            "Banner",
-        ]]
+        data = [
+            [
+                "HostID",
+                "Total Users",
+                "Host",
+                "Port",
+                "Banner",
+            ]
+        ]
 
         for h in hosts:
             total_users = self.db.get_loggedin_relations(host_id=h[0])
-            data.append([
-                h[0],
-                str(len(total_users)) + " User(s)",
-                h[1],
-                h[2],
-                h[3],
-            ])
+            data.append(
+                [
+                    h[0],
+                    str(len(total_users)) + " User(s)",
+                    h[1],
+                    h[2],
+                    h[3],
+                ]
+            )
```

Comment:
I actually disagree with this one, though I Think I will undo this formatting (it got auto-formatted). For this, its really useful to be able to see how many there are, because they need to have the same amount in each or it breaks the output. If they are in a one-liner it makes it much harder to know if you have the right amount of columns.

Reply:
Fine to me 👍